index.html

```diff
@@ -0,0 +1,176 @@
+        <div class="contents">
+          <h2>Tonic</h2>
+           <ul>
+             <li class="brand">CANOPY</li>
+             <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+             <li><img src="img/Counter.png" alt="counter"> 2015</li>
+            </ul>
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <ul class="li2">
+                 <li>html</li>
+                 <li>css</li>
+                 <li>javaScript</li>
+               </ul>
+                <button type="submit">See Project</button>
+           </div>
+        </article>
+      </aside>   
+ </div>
+  <!-- next page -->
+  <div class="continer">
+    <aside class="grid1">
+     <article>
+        <img src="img/Snapshoot(1).png" alt="Snapshoot Portfolio">
+        <div class="contents">
+          <h2>Multi-Post<br>Stories</h2>
+          <ul>
+            <li class="brand">CANOPY</li>
+            <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+            <li><img src="img/Counter.png" alt="counter"> 2015</li>
+          </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="li2">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+      <button type="submit">See Project</button>
+    </div>
+   </article>
+  </aside> 
+</div>
+<!--Next page -->
+  <div class="continer">
+    <aside class="grid1">
+      <article>
+       <img src="img/Snapshoot(2).png" alt="Snapshoot Portfolio">
+        <div class="contents">
+         <h2>Tonic</h2>
+         <ul>
+           <li class="brand">CANOPY</li>
+           <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+           <li><img src="img/Counter.png" alt="counter"> 2015</li>
+         </ul>
+         <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+         <ul class="li2">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+         </ul>
+         <button type="submit">See Project</button>
+       </div>
+    </article>
+   </aside>  
+ </div>
+<!--Next page -->
+  <div class="continer">
+    <aside class="grid1">
+      <article>
+      <img src="img/Snapshoot(3).png" alt="Snapshoot Portfolio">
+    
+    <div class="contents">
+      <h2>Multi-Post<br>Stories</h2>
+
+      <ul>
+        <li class="brand">CANOPY</li>
+        <li><img src="img/Counter.png" alt="counter"> Back End Dev</li>
+        <li><img src="img/Counter.png" alt="counter"> 2015</li>
+      </ul>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul class="li2">
+        <li>html</li>
+        <li>css</li>
+        <li>javaScript</li>
+      </ul>
+      <button type="submit">See Project</button>
+    </div>
+    </article>
+    </aside> 
+  </div>
```

Comment:
- [ ] Make sure to put all of your project cards inside a `<section>` tag to make use of semantic HTML

Reply:
@JuliCarracedo  Hello! If I do so there would be a linter error because it can't accept a section without having a general heading. you can see it on commit "update after checking".
what should I do now I have to submit it again without bring the changes that you mentioned?